client/src/app/components/CreateProject/Form.js

```diff
@@ -104,7 +104,7 @@ class Form extends React.Component {
                 "Sass"
               ]}
               maxAllowed={5}
-              default="React"
+              isDefault="React"
```

Comment:
The name 'default' was the right one. 'isDefault' is for a Boolean, which is not in this case

Reply:
I couldnt use default because it is a key word